ocs_ci/deployment/install_ocp_on_rhel.py

```diff
@@ -214,19 +220,25 @@ def prepare_rhel_nodes(self):
                 node, self.pod_ssh_key_pem, cmd, user=constants.VM_RHEL_USER
             )
 
-            # copy ops-mirror.pem
-            self.rhelpod.copy_to_server(
-                node,
-                self.pod_ssh_key_pem,
-                os.path.join(constants.PEM_PATH, constants.OCP_PEM),
-                constants.RHEL_TMP_PATH,
-                user=constants.VM_RHEL_USER,
-            )
-            pem_path_in_rhel = os.path.join(constants.RHEL_TMP_PATH, constants.OCP_PEM)
-            cmd = f"sudo cp {pem_path_in_rhel} {constants.PEM_PATH}"
-            self.rhelpod.exec_cmd_on_node(
-                node, self.pod_ssh_key_pem, cmd, user=constants.VM_RHEL_USER
-            )
+            for file_name in (
+                constants.MIRROR_OPENSHIFT_USER_FILE,
+                constants.MIRROR_OPENSHIFT_PASSWORD_FILE,
+            ):
+                self.rhelpod.copy_to_server(
+                    node,
+                    self.pod_ssh_key_pem,
+                    os.path.join(constants.YUM_VARS_PATH, file_name),
+                    os.path.join(constants.RHEL_TMP_PATH, file_name),
+                    user=self.rhel_worker_user,
```

Comment:
self.rhel_worker_user doesn't exist in this module

Reply:
Good catch! ... fixed in 0442ac3.